### ISSUE TITLE
Check site coordinates for amphibian data outside Rwanda

### ISSUE THREAD

**Issue by helendallas** [mmindje/Data-management-FRC-RBIS#29]
@mmindje @Thacien1 Please can you check the data falling outside Rwanda. The update on upload file so that it can be reingested. 

Here is the link to the sites outside Rwanda - amphibians
https://rbis.ur.ac.rw/map/#site-detail/taxon=&search=&siteId=&collector=&category=&yearFrom=&yearTo=&months=&boundary=&userBoundary=&referenceCategory=&spatialFilter=&reference=&endemic=&conservationStatus=[]&modules=8&validated=&sourceCollection=[%22RBIS%22]&abioticData=&ecologicalCategory=&rank=&siteIdOpen=&orderBy=name&polygon=[[29.494119286537167,-1.4401695324158794],[29.4921612739563,-1.442598844861422],[29.493083953857422,-1.4446581275522732],[29.495041966438297,-1.4449530767432464],[29.49638307094574,-1.4405985502547054],[29.49550867080688,-1.439681524526037],[29.494119286537167,-1.4401695324158794]]

**Comment by Thacien1** [comment]
> @mmindje @Thacien1 Please can you check the data falling outside Rwanda. The update on upload file so that it can be reingested.
> 
> Here is the link to the sites outside Rwanda - amphibians
> [https://rbis.ur.ac.rw/map/#site-detail/taxon=&search=&siteId=&collector=&category=&yearFrom=&yearTo=&months=&boundary=&userBoundary=&referenceCategory=&spatialFilter=&reference=&endemic=&conservationStatus=[]&modules=8&validated=&sourceCollection=[%22RBIS%22]&abioticData=&ecologicalCategory=&rank=&siteIdOpen=&orderBy=name&polygon=[[29.494119286537167,-1.4401695324158794],[29.4921612739563,-1.442598844861422],[29.493083953857422,-1.4446581275522732],[29.495041966438297,-1.4449530767432464],[29.49638307094574,-1.4405985502547054],[29.49550867080688,-1.439681524526037],[29.494119286537167,-1.4401695324158794]]](https://rbis.ur.ac.rw/map/#site-detail/taxon=&search=&siteId=&collector=&category=&yearFrom=&yearTo=&months=&boundary=&userBoundary=&referenceCategory=&spatialFilter=&reference=&endemic=&conservationStatus=%5B%5D&modules=8&validated=&sourceCollection=%5B%22RBIS%22%5D&abioticData=&ecologicalCategory=&rank=&siteIdOpen=&orderBy=name&polygon=%5B%5B29.494119286537167,-1.4401695324158794%5D,%5B29.4921612739563,-1.442598844861422%5D,%5B29.493083953857422,-1.4446581275522732%5D,%5B29.495041966438297,-1.4449530767432464%5D,%5B29.49638307094574,-1.4405985502547054%5D,%5B29.49550867080688,-1.439681524526037%5D,%5B29.494119286537167,-1.4401695324158794%5D%5D)

Thank you @helendallas . We will sort this out as soon as possible.

**Comment by mmindje** [comment]
Thank you @helendallas 

I am working my team to address the issue and have an update of the data
Regards

**Comment by helendallas** [comment]
Moved from #32 
Dear @helendallas and @mmindje

Here is an updated amphibians occurrence data file after removing the coordinates which fall outside the Rwandan boundaries. We suggest that it can be ingested into RBIS to replace the existing one.
RBIS Amphibian Data 2020_11_25 Final for RBIS V1.xlsx

Thank you

**Comment by helendallas** [comment]
@Thacien1 I have re-ingested these data to RBIS.  Please can you make sure that all the changes to coordinates and dates (done previously) are also taken over to the Amphibian Data Occurrence file that includes all occurrence records. I believe it is important to have the final complete and correct data file for each taxon group.

Once done, please can you share with me thanks

**Comment by Thacien1** [comment]
> @Thacien1 I have re-ingested these data to RBIS. Please can you make sure that all the changes to coordinates and dates (done previously) are also taken over to the Amphibian Data Occurrence file that includes all occurrence records. I believe it is important to have the final complete and correct data file for each taxon group.
> 
> Once done, please can you share with me thanks

Thank you so much @helendallas  . I was checking our amphibian site distribution on the RBIS web and found some data that fall outside of Rwanda. I think that we have duplicates files of amphibian occurrence data. Did Dimas delete the wrong file before you ingested the updated file? Thank you so much

**Comment by helendallas** [comment]
The process is you update the incorrect data for each occurrence records, making sure that the original UUID is used. Then when the data is reingested, the incorrect data are overwritten with the new data. Did you use the same UUID?

**Comment by helendallas** [comment]
@Thacien1 Have you checked the data for duplicates? As per comment above if the same UUID was used then the data should of been overwritten not duplicated

**Comment by Thacien1** [comment]
Dear @helendallas ,
What we did to amphibian occurrence data file is to delete all data which fall outside of Rwanda (data collected in Ngezi and Gasindikira wetlands). We did not changed and UUID for the remaining species occurrence. We suggest that Dimas can delete the existing file which has these two wetlands inside and then, we can upload the cleaned file. 

Here is our cleaned file.
[RBIS.Amphibian.Data.2020_11_25.Final.for.RBIS.V1.xlsx](https://github.com/mmindje/Data-management-FRC-RBIS/files/5635051/RBIS.Amphibian.Data.2020_11_25.Final.for.RBIS.V1.xlsx)

Thank you so much
